app/cards_routes.py

```diff
@@ -0,0 +1,58 @@
+from flask import Blueprint, request, jsonify, make_response
+from app import db
+from app.models.card import Card
+from app.routes_helpers import validate_model
+
+cards_bp = Blueprint("cards", __name__, url_prefix="/cards")
+
+@cards_bp.route("", methods=["POST"])
+def create_card():
```

Comment:
👀 We shouldn't be able to create a Card that _doesn't_ belong to a Board. Consider using the nested POST under Board to create a new Card which belongs to that Board, rather than associating an existing Card to an existing Board. That approach was slightly peculiar to Task List, which allowed Tasks to exist that weren't part of an overarching Goal.